web/workflow_modular.Rmd

```diff
@@ -0,0 +1,57 @@
+---
+title: "PEcAn Modular Workflow"
+author: "Aritra Dey"
+date: "`r Sys.Date()`"
+output: html_document
+---
+
+```{r libraries}
```

Comment:
Since this is a RMarkdown file, can you maybe add some text about what each section does. This will help for novice users to understand why this specific function is needed.

Reply:
Maybe add a short description and a link to the documentation.